custom/conf/app.example.ini

```diff
@@ -1038,7 +1038,7 @@ PATH =
 ;DEFAULT_THEME = gitea
 ;;
 ;; All available themes. Allow users select personalized themes regardless of the value of `DEFAULT_THEME`.
-;THEMES = gitea,arc-green
+;THEMES = gitea
```

Comment:
I think restricting to one default theme instead of two is a bad idea, especially if there is no way to force one or the other.
I could understand adding this as a third option, but overriding the current behavior in this way is in my opinion not a good idea.
I can (fore)see several problems with this implementation:
1) users can choose their theme themselves. The current implementation will interfere with this mechanism drastically.
2) I think it was explicitly mentioned that `arc-green` should not be seen as **THE** dark theme. It can be treated as a dark theme, but it is specifically not developed as a dark theme. There are other themes for that with more focus on being an actual dark theme.
3) Neither the name `gitea` nor `arc-green` indicate that they are a `light` or a `dark` theme.

What I think would be more relevant is to let users choose a theme themselves as light- or dark theme and then serve the theme based on the system preference if requested that way.

Reply:
I agree there should be additional option `auto`  that can be default

Reply:
I added a new default theme called `auto` which will be the `gitea` theme in light mode and `arc-green` in dark mode.